# Pensamiento geométrico para visualizar y argumentar

# Clase 3

**Caso #3:** Es un problema donde sólo nos dan un dato explícitamente, y otro dato no explícitamente.

* ***Ejemplo:*** Una persona ve la cima de una montaña cuando le apunta con un telescopio que forma $30^\circ$ con el piso. Se aleja $2\ \rm{km}$ y ahora apunta el telescopio de tal manera que mantenga un ángulo de $25^\circ$ con el piso. Si la altura del telescopio respecto del piso es de $0.5\ \rm{m}$, ¿podemos saber la altura de la montaña?

<img src="montana.png" align="middle" width="900">

Nuestro propósito es calcular $x$.

$$ \tan 30^\circ=\dfrac{x}{d} $$

$$ \tan 25^\circ=\dfrac{x}{d+2} $$

In [2]:
tand(30)

0.5773502691896258

In [3]:
tand(25)

0.46630765815499864

$$ 0.577=\dfrac{x}{d} $$

$$ 0.577d=x $$

$$ d=\dfrac{x}{0.577} $$

$$ 0.466=\dfrac{x}{d+2} $$

$$ 0.466=\dfrac{x}{\frac{x}{0.577}+2} $$

$$ 0.466\left(\dfrac{x}{0.577}+2\right)=x $$

$$ \dfrac{0.466x}{0.577}+2\cdot0.466=x $$

$$ \dfrac{0.466x}{0.577}+2\cdot0.466=x $$

$$ 0.8076x+0.932=x $$

$$ 0.8076x=x-0.932 $$

$$ 0.8076x-x=-0.932 $$

$$ -0.1924x=-0.932 $$

$$ x=\dfrac{-0.932}{-0.1924}=4.844 $$

4.844 km=4844m

Altura=4844.5 m

## El área de cualquier triángulo

### La altura de cualquier triángulo

<img src="triangulo.png" align="middle" width="300">

<img src="triangulo_altura.png" align="middle" width="300">

*La altura siempre es perpendicular a la base.*

**Para el triángulo (rectángulo) $ \Delta ABD $:**

$$ \sin A=\dfrac{h}{c} $$

$$ h=c\;\sin A $$

**Para el triángulo (rectángulo) $\Delta BCD$:**

$$ \sin C=\dfrac{h}{a} $$

$$ h=a\;\sin C $$

*En general tendremos que la altura de cualquier triángulo (perpendicular a algún cateto al que llamaremos "base") es el producto de un cateto (que no sea la base) y el seno del ángulo que forma dicho cateto con la base.*

### El área  conociendo dos catetos y el ángulo entre ellos

<img src="triangulo.png" align="middle" width="300">

*"Para cualquier triángulo el área es la mitad del producto de dos catetos y el seno del ángulo entre estos dos catetos."*

$$ \text{Area}=\dfrac{ab\sin C}{2}=\dfrac{bc\sin A}{2}=\dfrac{ac\sin B}{2} $$

### El área conociendo tres lados (Fórmula de Herón)

<img src="triangulo.png" align="middle" width="300">

$$ \text{Area}=\sqrt {s\left(s-a\right)\left(s-b\right)\left(s-c\right)}\ ,$$

donde $s=\dfrac{a+b+c}{2}=\dfrac{P}{2}$ es el semiperímetro (la mitad del perímetro).

* ***Ejemplo:*** Considera un triángulo oblicuángulo con $a=1200\ mm$, $c=700\ mm$, y $B=108^\circ$. Calcula su área.

<img src="triangulo.png" align="middle" width="300">

* Datos:
    * $a=1200\ mm$ 
    * $c=700\ mm$
    * $B=108^\circ$
    * $\text{Area}=?$

* Fórmulas:

$$ \text{Area}=\dfrac{ac\sin B}{2} $$

* Sustitución:

$$ \text{Area}=\dfrac{(1200)(700)\sin(108^\circ)}{2}=399443.74\ mm^2 $$

In [4]:
1200*700*sind(108)/2

399443.7368439645

## Triángulos oblicuángulos

### Ley de senos

Ésta nos sirve cuando:
* conocemos dos lados y un ángulo opuesto, y queremos conocer el otro ángulo opuesto
* conocemos dos ángulos y un lado opuesto, y queremos conocer el otro lado opuesto

$$ \dfrac{\sin A}{a}=\dfrac{\sin B}{b}=\dfrac{\sin C}{c} $$

*"La ley de senos me dice que el cociente del seno de un ángulo y su cateto opuesto es equivalente al seno de otro ángulo y su cateto opuesto"*

<img src="triangulo.png" align="middle" width="300">

### Ley de cosenos

Ésta nos sirve cuando:
* conocemos los tres lados y queremos conocer el ángulo opuesto a cualquiera de los lados
* conocemos dos lados y el ángulo entre estos lados, y queremos conocer el lado opuesto a este ángulo.

*El cuadrado de un lado es igual a la diferencia de la suma de los cuadrados de los otros lados y el doble del producto de los otros lados con el coseno del ángulo opuesto al lado inicial.*

$$ a^2=b^2+c^2-2bc\cos A $$

$$ b^2=a^2+c^2-2ac\cos B $$

$$ c^2=a^2+b^2-2ab\cos C $$

<img src="triangulo.png" align="middle" width="300">

* ***Ejemplo:*** Considera un triángulo oblicuángulo con $a=4$, $b=5$, y $B=30^\circ$.
    
    a) Encuentra $c$.

<img src="triangulo.png" align="middle" width="300">

* Datos:
     * $a=4$
     * $b=5$
     * $B=30^\circ$
     * $c=?$

* Fórmulas:

$$ c^2=a^2+b^2-2ab\,\cos C $$

$$ c=\sqrt{a^2+b^2-2ab\,\cos C} $$

* Sustitución:

$$ c=\sqrt{4^2+5^2-2(4)(5)\,\cos(30^\circ)} $$

In [5]:
√(4^2+5^2-2*4*5*cosd(30))

2.5217025694206003

b) Encuentra $A$.

* Datos:
     * $a=4$
     * $b=5$
     * $B=30^\circ$
     * $c=2.5217$
     * $A=?$

<img src="triangulo.png" align="middle" width="300">

* Fórmulas:

$$ a^2=b^2+c^2-2bc\cos A $$

$$ a^2-b^2-c^2=-2bc\cos A $$

$$ \dfrac{a^2-b^2-c^2}{-2bc}=\cos A $$

$$ \cos^{-1}\left(\dfrac{a^2-b^2-c^2}{-2bc}\right)= A $$

$$ A=\cos^{-1}\left(\dfrac{a^2-b^2-c^2}{-2bc}\right) $$

* Sustitución:

$$ A=\cos^{-1}\left(\dfrac{4^2-5^2-2.5217^2}{-2(5)(2.5217)}\right)=52.48^\circ $$

In [6]:
acosd((4^2-5^2-2.5217^2)/(-2*5*2.5217))

52.477560778904575

c) Encuentra $C$.

* Datos:
     * $a=4$
     * $b=5$
     * $B=30^\circ$
     * $c=2.5217$
     * $A=52.48^\circ$
     * $C=97.52^\circ$

# Tarea moral

* ***Ejemplo:*** Considera un triángulo oblicuángulo con $a=1200$, $c=700$, y $B=108^\circ$.
    
a) Encuentra $b$.

* Fórmulas:

* Sustitución:

b) Encuentra $A$.

c) Encuentra $C$.

## Círculos II

<img src="circ1.png" align="middle" width="300">

<img src="circ2.png" align="middle" width="300">

### Longitud de cuerda

<img src="cuerda.png" align="middle" width="300">

$$ \ell=2R\sin\left(\dfrac{\theta}{2}\right) $$

$$ \ell=2R\sin\left(\dfrac{\pi t}{360}\right) $$

* $\ell$ : longitud de cuerda
* $R$ : radio
* $\theta$ : ángulo (radianes)
* $t$ : ángulo (grados)

### Segmento circular

<img src="segmento.png" align="middle" width="300">

$$ A=\dfrac{\theta R^2}{2}-\dfrac{R^2\sin\theta}{2} $$

$$ A=\dfrac{R^2}{2}(\theta-\sin\theta) $$

$$ A=\dfrac{R^2}{2}\left[\dfrac{\pi t}{180}-\sin\left(\dfrac{\pi t}{180}\right)\right] $$

## Situación: El arte Burkina

**En un pueblo africano llamado Tiebele en Burkina Faso la corte real se aloja en pequeñas estructuras de adobe (arcilla y arena), que están decoradas con pinturas naturales que forman patrones geométricos. El propósito, además de estético, es para diferenciarlas de las casas de la gente común.**

**Tlahuilmalitzin es una artesana mexicana la cual admira el arte de cualquier región. Y quiere pintar una pared con un diseño parecido al de Burkina Faso. Pero ha notado que el tipo de figuras que adornan las paredes en Burkina son sencillas. Lo único que tiene que hacer es decidir su diseño de acuerdo con su presupuesto.**

<img src="extra1-1.png" align="middle" width="300">

**1. Dado que los anillos están separados a la misma distancia, Tlahuilmalitzin se pregunta si el área gris que se muestra en la siguiente figura, es: [1 punto]**

a)  $\dfrac{1}{2}$ de área del círculo mayor

b)  $\dfrac{3}{4}$ de área del círculo mayor

c)  $\dfrac{3}{8}$ de área del círculo mayor

d)  $\dfrac{5}{8}$ de área del círculo mayor

**Solución:** 

**2. Tlahuilmalitzin hizo otro diseño con las mismas circunferencias del modelo anterior, donde el punto $A$ está sobre la bisectriz del $\angle CBE$ , si $BC$ mide $1\ \rm{dm}$, el área de la estrella es: [2 puntos]**

<img src="extra1-2.png" align="middle" width="300">

a)  $\dfrac{\sqrt{2}}{2}\ \rm{dm}^2$

b)  $\sqrt{2}\ \rm{dm}^2$

c)  $2\sqrt{2}\ \rm{dm}^2$

d)  $\dfrac{\sqrt{2}}{4}\ \rm{dm}^2$

**Solución:** 

**3. Tlahuilmalitzin se pregunta ¿cómo son las áreas grises de ambas figuras? [2 puntos]**

<img src="extra1-3.png" align="middle" width="300">

a)  El área de los anillos es la mitad que el área de la estrella.

b)  El área de los anillos es mayor que la mitad del área de la estrella.

c)  El área de los anillos es igual que el área de la estrella.

d)  El área de los anillos es menor que la mitad del área de la estrella.

**Solución:** 

**4. Tlahuilmalitzin quiere adornar las paredes exteriores de su casa que construirá en Cuernavaca al estilo de las casas de Burkina Faso, ella tiene dos diseños combinando el de los anillos y las estrellas. ¿Cuál es el área en $\rm{m}^2$ que ocupan todos los añillos de ambos diseños? [3 puntos]**

<img src="extra1-4.png" align="middle" width="500">

<img src="extra1-5.png" align="middle" width="500">

a)  $\dfrac{5\pi}{4}\ \rm{m}^2$

b)  $\dfrac{13\pi}{80}\ \rm{m}^2$

c)  $\dfrac{13\pi}{5}\ \rm{m}^2$

d)  $\dfrac{\pi}{8}\ \rm{m}^2$

**Solución:** 

## Situación: Satélites artificiales

**Un satélite artificial es un ingenio enviado en una lanzadera espacial que se mantiene en órbita alrededor de cuerpos del espacio. Los satélites artificiales orbitan alrededor de satélites naturales, asteroides o planetas. Tras su vida útil, pueden quedar orbitando como basura espacial, o pueden desintegrarse reingresando en la atmósfera (cosa que ocurre solamente si su órbita es de poca altura).**

**Fuente: https://es.wikipedia.org/wiki/Sat%C3%A9lite_artificial**

a)  $\dfrac{40000}{\pi}$

b)  $\dfrac{20000}{\pi}$

c)  $40000\pi$

d)  $20000\pi$

**Solución:** 

**2. Si un satélite orbita circularmente a $2000\ \rm{km}$ de la superficie terrestre, ¿cuántos kilómetros mide la circunferencia de su órbita? [2 puntos]**


a)  $40000+4000\pi$

b)  $40000-4000\pi$

c)  $\dfrac{20000+2000\pi}{\pi}$

d)  $\dfrac{20000-2000\pi}{\pi}$

***Solución:***


**3. Si cada día un satélite recorre un ángulo de $\dfrac{\pi}{12}\ \rm{rad}$ , ¿cuántos días transcurren para que de una vuelta completa a la Tierra? [2 puntos]**

a)  $24$

b)  $12$

c)  $\dfrac{1}{24}$

d)  $\dfrac{1}{12}$

***Solución:***


**4.  Se planea lanzar un satélite a una altura de $1,000\ \rm{km}$ sobre la superficie terrestre, si este da una vuelta a la Tierra en $10$ días, ¿cuántos kilómetros sobre la superficie terrestre recorrerá cada día el satélite? [3 puntos]**

a)  $4000+200\pi$

b)  $200\pi$

c)  $\dfrac{720000+36000\pi}{\pi}$

d)  $36000$

***Solución:***


## Situación: Funámbulo

Un funámbulo o funambulista es un equilibrista que camina sobre un alambre o cuerda tensada.

Paul, que es uno de estos acróbatas, realiza un espectáculo de equilibrio a una altura de $5$ metros sobre el nivel del suelo, caminando sobre un cable de acero tenso entre los puntos $B$ y $C$ de las paredes de dos edificios separadas por $10$ metros, como se puede ver en la figura. Mientras camina del punto $B$ al punto $C$, lo sigue un reflector que se encuentra a $15$ metros de altura, en el punto $R$ de la figura, que provoca que genere una sombra en el piso.

<img src="extra4-1.png" align="middle" width="200">

<img src="extra_sol1.png" align="middle" width="300">

**1. Cuando Paul ha recorrido la mitad del cable, la distancia $\overline{AS}$ que ha recorrido su sombra, redondeada a una décima, es**

**Solución:** 

**2. Cuando el punto sombra $S$ del piso, alcanza la base del edificio en el punto $D$, la distancia que Paul ha recorrido desde el punto $B$ es**

**Solución:** 

**3. Cuando el punto sombra $S$ del piso llega al punto $D$, en ese instante se apaga el proyector que está en el punto $R$ y se prende otro proyector en el punto $A$ del piso y también provoca el punto $P$ de apoyo de Paul un punto sombra $S^{\prime}$, pero ahora sobre la pared $DG$ ¿A qué altura de la pared $DG$ (medida desde el piso), se encuentra el punto sombra en el instante en que se prende el proyector que se encuentra en $A$?**

**Solución:** 

## Situación: Estrellas

**1. El lado de un triángulo equilátero mide $x$ unidades. Su área, en unidades cuadradas, es (2 puntos)**

<img src="equi.png" align="middle" width="200">

a)  $\dfrac{3x^2}{2}$

b)  $\dfrac{\sqrt{3}x^2}{4}$

c)  $\dfrac{x^2}{2}$

d)  $\dfrac{\sqrt{3}x^2}{2}$

**Solución:** 

<img src="extra2-1.png" align="middle" width="200">

a)  $\dfrac{1}{3}$

b)  $\dfrac{1}{8}$

c)  $\dfrac{1}{4}$

d)  $\dfrac{1}{9}$

**Solución:** 

**3. En la siguiente figura se muestra un segmento $AB$, cuya longitud vale $a$ unidades. El área sombreada, en unidades cuadradas, es (2 puntos)**

<img src="extra2-2.png" align="middle" width="200">

a)  $\dfrac{\sqrt{3}a^2}{3}$

b)  $\dfrac{2\sqrt{3}a^2}{3}$

c)  $\dfrac{\sqrt{3}a^2}{4}$

d)  $\dfrac{3\sqrt{3}a^2}{4}$

**Solución:** 

**4. El segmento $AB$ de la siguiente figura tiene una longitud $a$. Entonces el área de la figura sombreada es (2 puntos)**

<img src="extra2-3.png" align="middle" width="200">

a)  $\dfrac{10\sqrt{3}a^2}{27}$

b)  $\dfrac{14\sqrt{3}a^2}{27}$

c)  $\dfrac{5\sqrt{3}a^2}{9}$

d)  $\dfrac{7\sqrt{3}a^2}{9}$

**Solución:** 